clib/dpd.js

```diff
@@ -2,9 +2,31 @@
 
   if (!window._dpd) window._dpd = {};
 
-  var root = window.location.protocol + '//' + window.location.hostname;
-  if (window.location.port !== '') {
-    root += ':' + window.location.port;
+  var root = dpdScriptElement().getAttribute('src')
+      .replace(/(.*)(\/.*\.js)$/, '$1')
+      .replace(/^[^\/]*.js$/, '');
+
```

Comment:
Good catch. Actually, I'm thinking a polyfill for document.currentScript might be the best way to go here...this looks promising: https://github.com/JamesMGreene/document.currentScript